source/TimePicker.js

```diff
@@ -479,10 +479,24 @@
 				default:
 					break;
 				}
-	
 			}
 	
 			this.inherited(arguments);
+
+			// in case of timePicker, timeWrapper enclose hour and minute pickers
+			// so, we re-organize this.pickers
+			var pickers = this.pickers;
```

Comment:
It seems a little awkward that we're allowing the super method to naïvely retrieve the client controls and assign that to `this.pickers`, only for us to reassign that to an array we build here. Also, I'd want to prevent unnecessarily looping through the controls when we have enough knowledge from the operations above. Instead, I'm wondering if we can build the array in the above switch statement (we would need to modify `wrapComponent` to return the created component, which should only be additive and wouldn't be regressive), and modify `DateTimePickerBase`'s `setupPickers` method to something like this:
```
this.pickers = this.pickers || this.getClientControls();
```
Any thoughts on this? I don't see any obvious cases where `this.pickers` would be incorrectly truthy when we run `DateTimePickerBase`'s `setupPickers` method, as this array will always be regenerated (in the case of `TimePicker`) or `null` (in the case of `DatePicker` via `reset`).